### DISCUSSION TITLE
the new client version is waiting long time for connection

### DISCUSSION THREAD

**Discussion by wxdjf0212** [rustdesk/rustdesk#9208] [General]
V1.2.3-2 connection speed is very fast, almost do not wait（After clicking, a password input box will appear immediately）
but the new version,after clicking,must wait a long time to show password input box
pls tell me why？how to change this problem？
（The above issues are in the same server）

**Comment by xlionjuan** [comment]
https://github.com/rustdesk/rustdesk/blob/master/src%2Flang%2Fen.rs#L202

**Reply by wxdjf0212** [reply]
I have tried both IP and domain, but the results are the same,still long time to connecting.

**Reply by xlionjuan** [reply]
@wxdjf0212 Force use Relay.

**Reply by wxdjf0212** [reply]
@xlionjuan I am using a self built API server。

**Reply by xlionjuan** [reply]
@wxdjf0212 3rd Party?


Read their README

**Comment by rustdesk** [comment]
#763
